https://gitlab.gnome.org/GNOME/pango/merge_requests/161

From b73284747ac937e31dc9191f84ed6e24284c88ee Mon Sep 17 00:00:00 2001
From: Chun-wei Fan <fanchunwei@src.gnome.org>
Date: Mon, 25 Nov 2019 14:58:32 +0800
Subject: [PATCH] tests/test-break.c: Only test Thai breaks with libthai
 available

The four.break test data file involves the Thai language, meaning that
this test will fail unless we have libthai.  Skip this test unless
libthai is found.
---
 tests/test-break.c | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/tests/test-break.c b/tests/test-break.c
index 8549b678..47fa3009 100644
--- a/tests/test-break.c
+++ b/tests/test-break.c
@@ -305,6 +305,12 @@ main (int argc, char *argv[])
       if (!strstr (name, "break"))
         continue;
 
+#ifndef HAVE_LIBTHAI
+      /* four.break involves Thai, so only test it when we have libthai */
+      if (strstr (name, "four.break"))
+        continue;
+#endif
+
       path = g_strdup_printf ("/break/%s", name);
       g_test_add_data_func_full (path, g_test_build_filename (G_TEST_DIST, "breaks", name, NULL),
                                  test_break, g_free);
-- 
2.24.1

